canonical-websites/www.canonical.com
# Upgrade /projects to v1

## Done

Upgrade /projects to v1

## QA

- Pull code
- Check markup at:
`templates/projects/index.html`
`templates/projects/directory.html`
- Run `./run serve --watch`
- Check page locally at http://0.0.0.0:8002/projects
- Check page locally at http://0.0.0.0:8002/projects/directory

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/29

## Demo

http://www.canonical.com-homepage-polish.demo.haus/projects
http://www.canonical.com-homepage-polish.demo.haus/projects/directory
